tests/mol-navigation/EPUB/package.opf

```diff
@@ -1,41 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<package xmlns="http://www.idpf.org/2007/opf"
-    xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
-    <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
-        <dc:coverage>Media Overlays</dc:coverage>
-        <dc:creator>Marisa DeMeglio</dc:creator>
-        <dc:date>2022-02-10</dc:date>
-        <dc:description>When navigating to another page, the media overlay playback position updates accordingly.</dc:description>
-        <dc:identifier id="pub-id">mol-navigation</dc:identifier>
-        <dc:language>en</dc:language>
-        <dc:title>Navigation with media overlays</dc:title>
-        <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
-        <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
-        <meta property="belongs-to-collection">should</meta>
```

Comment:
Note: I removed this entirely, because https://www.w3.org/TR/epub-rs-33/#sec-rsconf-navigation appears to be a MUST, not a SHOULD. Please let me know if I misunderstood something.

Reply:
Oh, wait: @iherman are all MO tests marked as SHOULD because the overall section is a SHOULD, even though these requirements are a MUST within the SHOULD?

I'm not sure what makes sense here:
a) From a CR perspective, we need to make sure all the MUSTs pass in at least 2 RSes, even if controlled by a SHOULD, to prove that the spec is implementable. This implies that all MO MUSTs should not be labeled as SHOULD with this attribute.

b) From an RS perspective, your system is compliant if it implements all the top-level MUSTs, and it may safely ignore this test.

IMO a) is more critical for us, and thus MO tests should not be marked as SHOULD. Thoughts?

Reply:
> Oh, wait: @iherman are all MO tests marked as SHOULD because the overall section is a SHOULD, even though these requirements are a MUST within the SHOULD?

yes, that was the reason.

> IMO a) is more critical for us, and thus MO tests should not be marked as SHOULD. Thoughts?

Yes, I guess that is indeed a better choice. A bit more dangerous, though: I am almost sure that some of the MUST statements will not have been implemented but, oh well, that is what testing is all about (and we will mark them as under-implemented).

Thanks for provoking these thoughts! 😀 But I would like to get at least @wareid or @dauwhe chime in on these.

Reply:
Restored for now (for consistency). Let's discuss in the next meeting, and if we decide these should not be SHOULDs, then we can pull them all with one PR.